resources/flakey-github-issue.template

```diff
@@ -0,0 +1,17 @@
+## Flaky Test
```

Comment:
An example of the UTs output

![image](https://user-images.githubusercontent.com/2871786/95223880-fe4e7700-07f1-11eb-8d6a-4aa5120b8ebe.png)